packages/copyable/src/Copyable.tsx

```diff
@@ -70,28 +57,62 @@ const codeStyle = css`
   height: 100%;
   width: 100%;
   padding-left: 12px;
-  border: 1px solid;
   border-radius: 6px;
   white-space: nowrap;
-  overflow: hidden;
+  overflow: auto;
   grid-column: 1/-1;
   grid-row: 1/2;
 `;
 
-const largeCodeStyle = css`
-  font-size: 15px;
+// Border is removed from the code component and added to the parent
+const codeStyleNoButton = css`
+  border: 0;
 `;
 
-const noButtonCodeStyle = css`
-  overflow: hidden;
+const labelNoButtonStyle = css`
+  font-size: 18px;
+  line-height: 24px;
 `;
 
-const codeStyleColor = (mode: Mode) => css`
-  color: ${colorSets[mode].code.text};
-  background-color: ${colorSets[mode].code.background};
-  border-color: ${colorSets[mode].code.border};
+const labelFontStyle: Record<BaseFontSize, string> = {
+  [BaseFontSize.Body1]: css`
+    font-size: ${typeScales.body1.fontSize}px;
+    line-height: ${typeScales.body1.lineHeight}px;
+  `,
+  [BaseFontSize.Body2]: css`
+    font-size: ${typeScales.body2.fontSize}px;
+    line-height: 20px; // Hardcoding because it does not match body2 lineHeight
+  `,
+};
```

Comment:
These styles should be in the `Label` component, no?

Reply:
They are in the `Label` component but the label component font size is based off of the global font size and doesn't accept a size prop so if the size prop in the copyable component is different than the global font size the label component will not recognize that so I had to duplicate the styles in here. I could export out the `Label` styles then that way I can reuse them in here.